https://issues.redhat.com/browse/ACM-5559 : add .yaml callout (#5085)
open-cluster-management/rhacm-docs: add-ons/cluster_wide_proxy.adoc

----
spec
  proxyConfig:
    httpProxy: "<proxy_not_secure>" <1>
    httpsProxy: "<proxy_secure>" <2>
    noProxy: "<no_proxy>" <3>
----
+
<1> Replace `proxy_not_secure` with the address of the proxy server for `http` requests. For example, use `http://192.168.123.145:3128`.
<2> Replace `proxy_secure` with the address of the proxy server for `https` requests. For example, use `https://192.168.123.145:3128`. 

> <3> Replace `no_proxy` with a comma delimited list of IP addresses, hostnames, and domain names where traffic will not be routed through the proxy. For example, use `.cluster.local,.svc,10.128.0.0/14,example.com`.

Reviewer: Suggest to remove "will" to maintain consistent, present tone. 

```suggestion
<3> Replace `no_proxy` with a comma delimited list of IP addresses, hostnames, and domain names where traffic is not routed through the proxy. For example, use `.cluster.local,.svc,10.128.0.0/14,example.com`.
```
